src/lib/pushDoc.js

```diff
@@ -32,6 +32,16 @@ module.exports = async function pushDoc(key, selectedVersion, dryRun, filepath,
   const slug = matter.data.slug || path.basename(filepath).replace(path.extname(filepath), '').toLowerCase();
   const hash = crypto.createHash('sha1').update(file).digest('hex');
 
+  let data = { body: matter.content, ...matter.data, lastUpdatedHash: hash };
+
+  if (type === 'custompages') {
+    if (filepath.endsWith('.html')) {
```

Comment:
Related to my comment in that other PR but we might want to drop case-sensitive checking on these extensions so we can allow `.HTML` through.

Reply:
Fixed this here: https://github.com/readmeio/rdme/pull/549/commits/706cdaf8ec5011d16164a51f8406bf4a7a20c53f